pytorch_lightning/lite/lite.py

```diff
@@ -40,10 +40,9 @@
     TrainingTypePlugin,
 )
 from pytorch_lightning.trainer.connectors.accelerator_connector import AcceleratorConnector
-from pytorch_lightning.trainer.data_loading import TrainerDataLoadingMixin
```

Comment:
reviewer: this is the main motivation of the PR, not having to import a Trainer "mixin" for Lite